mkt/api/serializers.py

```diff
@@ -70,7 +70,7 @@ class CarrierSerializer(serializers.Serializer):
 
 
 class RegionSerializer(CarrierSerializer):
-    default_currency = serializers.CharField()
+    pass
```

Comment:
I thought about doing that, but I preferred leaving them separate for now, I might tweak the region one soon. If we do merge them I'd prefer simply having `RegionOrCarrierSerializer` or something similar.